### PR TITLE
Style changes based on editorconfig recommendations

### PR DESCRIPTION
**Pull request by marknfawaz**
## Related issue

Closes: #48 

## Description
Layout updates to code for better maintainability

## Supplemental testing
N/A

## Additional context
N/A

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs [hunk 1]
```diff
@@ -67,28 +67,28 @@ public void Add(IEnumerable<Feature> features)
...
-            if (feature is CompiledFeature)
+            if (feature is CompiledFeature feature1)
             {
                 if (ConfiguredFeatures.Any(f => f.Name == feature.Name))
                 {
                     throw new ArgumentException(string.Format(FeatureNameConflictTemplate, feature.Name, nameof(ConfiguredFeature)));
                 }
 
-                CompiledFeatures.Add((CompiledFeature)feature);
+                CompiledFeatures.Add(feature1);
```
Comment (jonlouie) [inline src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs:77]: nit: can we rename `feature1` to `compiledFeature`?

File: src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs [hunk 2]
```diff
@@ -67,28 +67,28 @@ public void Add(IEnumerable<Feature> features)
...
-            else if (feature is ConfiguredFeature)
+            else if (feature is ConfiguredFeature feature2)
             {
                 if (CompiledFeatures.Any(f => f.Name == feature.Name))
                 {
                     throw new ArgumentException(string.Format(FeatureNameConflictTemplate, feature.Name, nameof(CompiledFeature)));
                 }
 
-                ConfiguredFeatures.Add((ConfiguredFeature)feature);
+                ConfiguredFeatures.Add(feature2);
```
Comment (jonlouie) [inline src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs:86]: nit: can we rename `feature2` to `configuredFeature`?

File: src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs (hunk 1 above)
Comment (marknfawaz) [reply src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs]: ok

File: src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs (hunk 2 above)
Comment (marknfawaz) [reply src/CTA.FeatureDetection.Common/Models/Features/FeatureSet.cs]: ok

**Review by jonlouie — APPROVED** [review]

**Review by jonlouie — APPROVED** [review]

**Pull request merged**